services/user/Registry/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[workspace]
+resolver = "2"
+members = ["service", "query", "plugin"]
+
+[workspace.package]
+version = "0.1.0"
+rust-version = "1.64"
+edition = "2021"
+publish = false
+repository = "https://github.com/gofractally/psibase"
+homepage = "https://psibase.io"
+
+[package]
+edition = "2021"
+name = "registry_package"
+version = "0.1.0"
+
+[package.metadata.psibase]
+package-name = "RegistryPackage"
```

Comment:
The package shouldn't have "Package" in the name. That would be like adding "Crate" into the name of rust crates.
I think you should call the package "AppRegistry" (regardless that the service is only called, "registry").